README.md

```diff
@@ -62,46 +73,62 @@ entering the system) and matrices containing growth and survival rates for each
 `FunctionalGroup` and `SizeClass` is needed:
 
 ```julia
-using CoralBlox: timestep
-
-# Mock recruits
-recruits::Vector{Float64} = [1e5, 2e5]
+using CoralBlox: linear_extension_scale_factors, max_projected_cover,
+using CoralBlox: timestep!, coral_cover
 
 # Mock linear extensions
 linear_extensions::Matrix{Float64} = [
-    15 25 35 25;
-    10 20 25 15;
+    0.004 0.025 0.1 0.0;
+    0.003 0.007 0.005 0.0;
+    0.002 0.004 0.01 0.0
 ]
 
 # Mock survival rate
-survival_rate::Matrix{Float64} = 1 .- [
-    0.05 0.05 0.03 0.01;
-    0.1 0.08 0.04 0.02
+survival_rate::Matrix{Float64} = [
+    0.5 0.6 0.65 0.8;
+    0.6 0.7 0.8 0.8;
+    0.5 0.8 0.8 0.8
 ]
 
-for tstep::Int64 in 2:n_timesteps
-    # Linear extension scale factor
-    lin_ext_scale_factors::Vector{Float64} = linear_extension_scale_factors(
-        C_cover[tstep, :, :],
-        available_area,
-        linear_extensions,
-        size_class_bounds,
-        habitable_max_projected_cover,
-    )
+# Caclulate maximum projected cover
+habitable_max_projected_cover = max_projected_cover(
+    linear_extensions,
+    size_class_bounds,
+    habitable_area
+)
+
+# Only apply linear extension scale factor when cover is above the scale_threshold
```

Comment:
For each of these statements, it would be useful to explain why in a single sentence.

Reply:
When you say "each of these statements" what are the other statements you are refering to?

Reply:
I'll mark them

Reply:
Turns out there were only two others? I thought I saw more, but there you go.